docs/style.css

```
@@ -0,0 +1,324 @@
+
+code-group *,
+code-block *,
+#content pre,
+#content pre *,
+mdx-content pre,
+mdx-content pre * {
+  min-width: 0 !important;
+  max-width: 100% !important;
+  width: auto !important;
+  white-space: pre-wrap !important;
+  word-break: break-word !important;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)